tools/cli/commands/changelog.js

```diff
@@ -213,6 +214,26 @@ export function changelogDefine( yargs ) {
 					async argv => {
 						await changelogArgs( argv );
 					}
+				)
+				// Squash subcommand.
+				.command(
+					'squash [project]',
+					'Squashes changelog projects',
+					yargSquash => {
+						yargSquash
+							.positional( 'project', {
+								describe: 'Project in the form of type/name, e.g. plugins/jetpack',
+								type: 'string',
+							} )
+							.option( 'readme', {
+								describe: 'Indicates if we only want to squash the readme and not the changelog.',
```

Comment:
The wording here and in the prompt implies that the readme will always be squashed, and the question is whether the changelog is also squashed or not. The actual behavior is that either the changelog or the readme is squashed, not both.

Either way may make sense, as far as I'm concerned. We just need the prompts and behavior to agree.